docs/source/technical_tutorials/search/collection_discovery.ipynb

```diff
@@ -5,30 +5,27 @@
    "id": "337f5e01-9946-465a-8657-e4153fbac541",
```

Comment:
<p><code>collection-search-only</code> this is awkward mouthful, not sure how best to break it up</p><p>"only a collection-search API" </p><p>You really assume people know what an API is ...</p><p> </p><p> </p><p>the paragraph with "The API uses the collection-search STAC API extension..." is a background note for advanced users and developers, move it down to a footnote somewhere.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/541/discussion/#comment-2625181389'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>